pkg/ibmcsidriver/controller_helper.go

```diff
@@ -99,10 +104,12 @@ func getVolumeParameters(logger *zap.Logger, req *csi.CreateVolumeRequest, confi
 	var err error
 	volume := &provider.Volume{}
 	volume.Name = &req.Name
+	var profileName string
 	for key, value := range req.GetParameters() {
 		switch key {
 		case Profile:
 			if utils.ListContainsSubstr(SupportedProfile, value) {
+				profileName = value
```

Comment:
why we can't use `volume.VPCVolume.Profile.Name` instead of creating new variable?

Reply:
@arahamad I had initially used that only but wanted to avoid multiple nil checks. I have now put generic nil check to avoid this. Thanks for the comments.